### ISSUE TITLE
Make the user explicit in the ssh command

### ISSUE THREAD

**Issue by dlespiau** [dlespiau/footloose#38]
At the moment we hardcode "root@nodeIP" as the ssh argument, we should probably make the user explicit and default to the current user, like the ssh command does, to avoid surprising the user:

```
$ footloose ssh root@nodes
```

This should also play better with the plan of letting users login with heir own id/gid as described in #11.

**Comment by ilmanzo** [comment]
Looking at it , I think it needs a bit of redesign, because the user home directory is involved also in the ssh key creation:

```
const initScript = `
set -e
rm -f /run/nologin
sshdir=/root/.ssh
mkdir $sshdir; chmod 700 $sshdir
touch $sshdir/authorized_keys; chmod 600 $sshdir/authorized_keys
`

```

and

```
publicKey, err := c.publicKey()
if err != nil {
	return err
}
if err := copy(name, publicKey, "/root/.ssh/authorized_keys"); err != nil {
	return err
}

```

**Comment by dlespiau** [comment]
yes, you are right! I think there are two different things:

1. Allow the creation of arbitrary users when creating the container (this idea wasn't really properly captured yet, filed https://github.com/dlespiau/footloose/issues/43)
2. The syntax of the `ssh` command (which is the subject of this issue and is a much smaller change). I believe the `ssh` command syntax should follow what the real ssh does:

```
$ footloose ssh node0
-> try to login with the current user

$ footloose ssh root@node0
-> try to login with the root user
```

Of course the first thing will not work until we can provision new users into the containers.

There is a detail that needs to be taken into consideration as well: it'd be great to be able to mirror the current user inside the container (name, id, gid) so we can mount part of the filesystem in the container and still end up with the right id/gid for the files we create from inside the container: https://github.com/dlespiau/footloose/issues/11.

**Comment by dlespiau** [comment]
Just to be clear, I think it's fine to just address the point 2. above separately from the more general problem of user creation.

**Comment by dlespiau** [comment]
We could also say that user provisioning is not the domain of machine creation and is best left to other tools such as ansible. We could still install the ssh key for the root user tough, to bootstrap the other tools.

That may be fair game.

**Comment by dlespiau** [comment]
Fixed by #44
